apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-preview-dropdown-selected.js

```diff
@@ -0,0 +1,30 @@
+import { __ } from '@wordpress/i18n';
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_template_part_detach_blocks`.
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export default () => ( {
+	id: 'wpcom-preview-dropdown-selected',
+	selector: '.components-menu-item__button',
+	type: 'click',
+	capture: true,
+	handler: ( _event, target ) => {
+		const mapTextToPreviewMode = {
+			[ __( 'Desktop' ) ]: 'desktop',
+			[ __( 'Desktop (50%)' ) ]: 'zoom-out',
+			[ __( 'Tablet' ) ]: 'tablet',
+			[ __( 'Mobile' ) ]: 'mobile',
+			[ `${ __( 'View site' ) }\n${ __( '(opens in a new tab)' ) }` ]: 'view-site',
```

Comment:
Can we include "Preview in new tab" as well to track all options that are available in the post editor?

<img width="280" alt="Screenshot 2024-08-29 at 09 29 05" src="https://github.com/user-attachments/assets/323e8a40-e8b1-499c-bb1b-6e7c6e2ab526">

Reply:
Added!